source/state_representation/README.md

```diff
@@ -130,7 +130,7 @@ StateRepresentation::CartesianTwist aVb("b", "a");
 StateRepresentation::CartesianTwist wVa = wPa + aVb;
 ```
 
-### Conversion between Cartesian state variables
+### Conversion between cartesian state variables
```

Comment:
Ah see Cartesian should stay uppercase because it's a proper name (like Newton, Euler...). Sorry if that wasn't clear in my original comment!

Reply:
It was. It is me the dumb dumb